misk-jdbc/src/main/kotlin/misk/jdbc/DataSourceService.kt

```diff
@@ -145,4 +175,15 @@ class DataSourceService @JvmOverloads constructor(
   override fun get(): DataSource {
     return dataSource
   }
+
+  // Copy pasta from PoolBase.initializeDataSource()
```

Comment:
Where is `PoolBase`? Probably worth formalizing the comment here since this function will live here long term.